django_drf_filepond/views.py

```diff
@@ -75,14 +90,15 @@ class ProcessView(APIView):
     # from FilePond.
     parser_classes = (MultiPartParser,)
     renderer_classes = (PlainTextRenderer,)
+    permission_classes = _import_permission_classes('POST_PROCESS')
 
     def post(self, request):
         LOG.debug('Filepond API: Process view POST called...')
 
         # Enforce that the upload location must be a sub-directory of
         # the project base directory
         # TODO: Check whether this is necessary - maybe add a security
-        # parameter that can be disabled to turn off this check if the
+        # parameter that can be disabled to turn off this check if th
```

Comment:
Sorry about this :)

Reply:
No worries, thanks for pulling these changes into your PR.